cfgov/jinja2/v1/_includes/organisms/filterable-list.html

```diff
@@ -0,0 +1,195 @@
+{# ==========================================================================
+
+   Filterable list
+
+   ==========================================================================
+
+   Description:
+
+   Create a filter list that includes controls and results when given:
+
+   value: Collection of field controls
+          and other settings to determine what's rendered.
+
+   value.no_posts_message: Message to show in "no posts" state.
+
+   value.no_posts_explanation: More info to show in "no posts" state.
+
+   value.form_type: What type of form this is.
+
+   value.categories.page_type: Page type of the filterable list control.
+
+   value.output_5050: Whether to render results as info-units or not.
+
+   value.link_image_and_heading: Whether to link the image and heading
+                                 in info-units.
+
+   filter_data.form: Django form that carries the fields to be rendered.
+
+   filter_data.page_set: Result set of posts from the filtered content.
+
+   ========================================================================== #}
+
+{% import 'organisms/filterable-list-controls.html' as filterable_list_controls with context %}
+{% import 'molecules/notification.html' as notification with context %}
+
+{# SHOW A NOTIFICATION IF THERE'S NOTHING TO FILTER. #}
+
+{% set has_results = filter_data.form.filterable_pages.first() %}
+
+{% if not has_results and value.no_posts_message %}
+    {{ notification.render(
+        'default',
+        true,
+        value.no_posts_message,
+        value.no_posts_explanation
+    ) }}
+{% elif has_results %}
+
+    <div class="o-filterable-list">
+        {% set form = filter_data.form %}
+        {% set posts = filter_data.page_set %}
+
+        {# ADD THE FILTER CONTROLS. #}
+
+        {% set fragment_id = 'o-filterable-list-controls' %}
+        {{ filterable_list_controls.render(value, form, fragment_id) }}
+
+        {# ADD THE NOTIFICATION, IF NEEDED. #}
+
+        {% set notification_opts = {
+            'type': 'default',
+            'is_visible': false,
+            'message': '',
+            'explanation': ''
+        } %}
+
+        {% for field in form %}
+            {% if field.errors %}
+                {% for error in field.errors %}
+                    {# The if statement here allows access to the
+                       notification_opts variable outside of the for loop.
+                    #}
+                    {% if notification_opts.update({
+                        'type': 'error',
+                        'is_visible': true,
+                        'message': notification_opts.message + field.label + ': ' + error + '<br>' | safe
+                    }) %}{%endif%}
+                {% endfor %}
+            {% endif %}
+        {% endfor %}
+        {% if 'filterable-list' in value.form_type and posts is defined %}
```

Comment:
Trivial: Consider adding a blank line to visually separate the `endfor` from the `if`.